docs/wfirst.rst

```diff
@@ -0,0 +1,96 @@
+
+What is different (and, for now, only available for the WFIRST WFI) is the support for field dependent aberrations. With as large a field of view as the WFI is designed to cover, there will be variation in the PSF from one end of the field of view to the other. WebbPSF's WFI model faithfully reproduces the field dependent aberrations calculated from the Goddard WFIRST team's simulated WFI design, leaving it to the user to decide if their use case requires accurate knowledge of very small variations in the PSF.
+
+.. admonition:: Quickstart IPython Notebook
+
+   This documentation is complemented by an `IPython Notebook format quickstart tutorial <http://nbviewer.ipython.org/github/mperrin/webbpsf/blob/master/notebooks/WebbPSF-WFIRST_Tutorial.ipynb>`_. Downloading and run that notebook to use the beta notebook GUI for the WFI model, and to explore code samples for common tasks interactively.
+
+Field dependence in the WFI model
+=================================
+
+Field points are specified in a WebbPSF calculation by selecting a detector and pixel coordinates within that detector. A newly instantiated WFI model already has a default detector and position. ::
+
+   >>> wfi.detector
+   'SCA01'
+   >>> wfi.detector_position
+   (2048, 2048)
+
+.. figure:: ./wfirst_figures/field_layout.png
+   :alt: The Wide Field Instrument's field of view, as projected on the sky.
+
+
+The WFI field of view is laid out as shown in the figure. To select a different detector, assign its name to the ``detector`` attribute::
+
+   >>> wfi.detector_list
+   ['SCA01', 'SCA02', 'SCA03', 'SCA04', 'SCA05', 'SCA06', 'SCA07', 'SCA08', 'SCA09', 'SCA10', 'SCA11', 'SCA12', 'SCA13', 'SCA14', 'SCA15', 'SCA16', 'SCA17', 'SCA18']
+   >>> wfi.detector = 'SCA03'
+
+The usable region of the 4096 by 4096 pixel detectors specified for the Wide Field Instrument will range from (4, 4) to (4092, 4092), accounting for the 4 pixel wide bands of reference pixels. To change the position at which to calculate a PSF, simply assign an (X, Y) tuple::
+
+   >>> wfi.detector_position = (4, 400)
+
+.. warning::
+
+   WebbPSF will not prevent you from setting an out of range detector position, but an error will be raised if you try to calculate a PSF with one. ::
+
+      >>> wfi.detector_position = (1, 1)
+      >>> wfi.calcPSF()
+      [ ... traceback omitted ... ]
+      RuntimeError: Attempted to get aberrations for an out-of-bounds field point
+
+The reference information available gives the field dependent aberrations in terms of Zernike polynomial coefficients from :math:`Z_1` to :math:`Z_{22}`. These coefficients are calculated for five field points on each of 18 detectors, each at 16 unique wavelengths providing coverage from 0.76 :math:`\mu m` to 2.0 :math:`\mu m` (that is, the entire wavelength range of the WFI).
+
```

Comment:
add statement about the code automatically interpolating between those five field points as appropriate.